metadata-ingestion/src/datahub/ingestion/source/unity/proxy.py

```diff
@@ -45,6 +46,8 @@
 
 logger: logging.Logger = logging.getLogger(__name__)
 
+MAX_WORKERS = 100
```

Comment:
this looks unused?

Reply:
Nice catch, was using this before I added the config param